README.md

```diff
@@ -78,6 +88,8 @@ def print_array(array)
 end
 ```
 
+* SNIPPET 6: 0(n) linear
```

Comment:
THis is actually O(n^2) because the while loop on the inside runs O(n) times and the .each loop also runs O(n) times.